frontend/src/lib/components/PasswordStrength.tsx

```diff
@@ -15,14 +15,12 @@ export default function PasswordStrength({
 
     return (
         <Tooltip title="Password strength">
-            <Progress
+            <LemonProgress
                 percent={passwordScore}
-                size="small"
-                strokeColor={
+                trackColor={
```

Comment:
I do believe `strokeColor` better represented what this means, because the track is the gray underlying bar. I think `strokeColor` or `rangeColor` would be clearer here. (E.g. Radix UI [calls these parts "track" and "range"](https://www.radix-ui.com/primitives/docs/components/slider))